### PR TITLE
#92 add data for permission

### PR DESCRIPTION
**Pull request by BorchBe99**
Added permissions of all Directors and list of all existing permissions

### CODE REVIEW

**Comment by LennartRukower** [comment]
Please link the corresponding issue in the description or via the "Linked issues"-tab.

File: server/src/members/membersRoutes.ts [hunk 1]
```diff
@@ -17,6 +17,8 @@ router.get("/", membersController.retrieveMemberList);
 router.get("/directors", membersController.retrieveDirectors);
 router.get("/current-directors", membersController.retrieveCurrentDirectors);
 router.get("/department-members", membersController.retrieveDepartmentMembers);
+router.get("/permissions-of-members", membersController.retrievePermissionsOfMembers);
```
Comment (LennartRukower) [inline server/src/members/membersRoutes.ts:20]: Retrieving permissions should not be possible for every member.
Only members with special permissions and roles should be able to retrieve permissions.
This route should be **restricted**!

File: server/src/members/membersRoutes.ts [hunk 2]
```diff
@@ -17,6 +17,8 @@ router.get("/", membersController.retrieveMemberList);
...
+router.get("/permissions", membersController.retrievePermissions);
```
Comment (LennartRukower) [inline server/src/members/membersRoutes.ts:21]: Retrieving permissions should not be possible for every member.
Only members with special permissions and roles should be able to retrieve permissions.
This route should be **restricted**!

**Review by LennartRukower — CHANGES_REQUESTED** [review]
Good pr, @BorchBe99 !
There are just a few minor issues.

File: server/src/members/membersController.ts [hunk 3]
```diff
@@ -191,7 +191,7 @@ export const retrieveDepartmentMembers = (req: Request, res: Response): void =>
 /**
  * Retrieves all current directors
  */
- export const retrieveCurrentDirectors = (req: Request, res: Response): void => {
+export const retrieveCurrentDirectors = (req: Request, res: Response): void => {
```
Comment (christophtrybek) [inline server/src/members/membersController.ts:194]: What happened here?

File: server/src/members/membersController.ts [hunk 4]
```diff
@@ -255,7 +255,7 @@ export const retrieveDirectors = (req: Request, res: Response): void => {
 /**
  * Retrieves the departments
  */
- export const retrieveDepartments = (req: Request, res: Response): void => {
+export const retrieveDepartments = (req: Request, res: Response): void => {
```
Comment (christophtrybek) [inline server/src/members/membersController.ts:258]: What happened here?

**Review by christophtrybek — CHANGES_REQUESTED** [review]
Routes need to be private, this is very important! Please consider indentation in future PRs

**Comment by LennartRukower** [comment]
Please add a label to this pr and a project/milestone.

File: server/src/members/membersController.ts (hunk 3 above)
Comment (BorchBe99) [reply server/src/members/membersController.ts:194]: Fixed indentation

File: server/src/members/membersController.ts (hunk 4 above)
Comment (BorchBe99) [reply server/src/members/membersController.ts:258]: Fixed indentation

**Review by LennartRukower — APPROVED** [review]
LGTM!

**Review by christophtrybek — APPROVED** [review]
LGTM

**Pull request merged**